html/browsers/windows/opener-cross-origin-manual.sub.html

```diff
@@ -0,0 +1,10 @@
+<ol>
+ <li><p>Clicking this link must navigate this page to example.com:
+ <a href=//www1.{{location[host]}}/html/browsers/windows/support-opener.html target=_blank>test</a>
+
+ <li><p>Clicking this link must open a new browsing context that is empty:
+ <a rel=noreferrer href=//www1.{{location[host]}}/html/browsers/windows/support-opener.html target=_blank>test</a>
+
+ <li><p>Clicking this link must navigate this page to example.com:
+ <a href=//www1.{{location[host]}}/html/browsers/windows/support-embed-opener.sub.html target=_blank>test</a>
```

Comment:
This doesn't look like it should work. That is, if you open `www1.whatever`, which frames `something-else.whatever`, then my intuition is that the cross-origin frame shouldn't be able to reach up into it's parent's opener, or close it's parent. Does that work in Firefox? I guess I'll have to run this in Chrome to see what it does.